File: TuristickaAgencija.Model/Rezervacija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public partial class Rezervacija
    {
        public int Id { get; set; }
        public string Ime { get; set; }
        public int? KorisnikId { get; set; }
        public int? PutovanjeId { get; set; }
        public DateTime DatumRezervacije { get; set; }
        public int BrojOsoba { get; set; }
        public string Status { get; set; }
        public string Napomena { get; set; }
    }
}

File: UI/turisticka_agencija_mobile/lib/models/Rezervacija.dart
import 'package:json_annotation/json_annotation.dart';

part 'Rezervacija.g.dart';

@JsonSerializable()
class Rezervacija {
    int brojOsoba;
    String datumRezervacije;
    int id;
    String ime;
    int korisnikId;
    String napomena;
    int putovanjeId;
    String status;

    Rezervacija({required this.brojOsoba,required this.datumRezervacije,required this.id,required this.ime,
        required this.korisnikId,required this.napomena,required this.putovanjeId,required this.status});

    factory Rezervacija.fromJson(Map<String, dynamic> json) => _$RezervacijaFromJson(json);

    Map<String, dynamic> toJson() => _$RezervacijaToJson(this);
}